src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const liElements = document.querySelectorAll('li');
+const ulElement = document.querySelector('ul');
+const arr = [...liElements];
+
+function toNumb(salary) {
+  return +salary.slice(1).replace(',', '');
+}
+
+const sortList = function(list) {
+  return list.sort((a, b) => {
+    return (toNumb(b.dataset.salary) - toNumb(a.dataset.salary));
+  });
+};
+
+const newLi = sortList(arr);
+
+[...liElements].forEach(el => el.remove());
+
+ulElement.insertAdjacentHTML('afterbegin', `
+    ${[...newLi].map(person => `
+      ${person.outerHTML}
+    `).join('')}
+`);
+
+function getEmployees(list) {
+  const arrayObj = [...list].map(person => {
```

Comment:
Map is a non-mutating array, so there is no need to create a copy.
Check yourself